graphql/resolve/mutation_rewriter.go

```diff
@@ -600,46 +602,48 @@ func (urw *UpdateRewriter) Rewrite(
 
 	upsertQuery := RewriteUpsertQueryFromMutation(m, authRw, MutationQueryVar, m.Name(), "")
 	srcUID := MutationQueryVarUID
-
-	if setArg == nil && delArg == nil {
+	objDel, okDelArg := delArg.(map[string]interface{})
```

Comment:
This statement is copy of line 345. I believe this statement could be written before line 344 to avoid statement on line 345. Similarly, for setArg.

Reply:
statement at line 345  is in different function.